app/views/spree/admin/orders/_form.html.haml

```diff
@@ -6,7 +6,8 @@
     = render 'spree/admin/orders/insufficient_stock_lines', insufficient_stock_lines: @order.insufficient_stock_lines
 
   = render :partial => "spree/admin/orders/shipment", :collection => @order.shipments, :locals => { :order => order }
-  = render partial: "spree/admin/orders/note", locals: { order: @order }
+  - unless @order.shipments.empty?
```

Comment:
I think we should better use 
```
if @order.shipments.present?
```
instead

Also, I'm wondering if this was a requirement of the original issue ?

Reply:
Updated both places with `present?`. It makes for cleaner code, thanks!
And yeah, this is a requirement, it satisfies num. 2 on https://github.com/openfoodfoundation/openfoodnetwork/issues/9409

Reply:
Also I'll get started on shifting all this to StimulusJs but I think I should work on it at a separate PR. So I will create an issue for that.

Reply:
Ok, found that point `#2`, thanks for pointing me that out. 
Event it's written, `hide it just like tracking information`, I don't think we should test for `@order.shipments` but more `order.line_items` or `@line_items` right? I think Note can be relevant on order that don't have any shipment. 

And as it's a requirement, I think we should add it into the `What should we test?` section. 

> Also I'll get started on shifting all this to StimulusJs but I think I should work on it at a separate PR. So I will create an issue for that.

I agree. Feel free to open an issue for that.

Reply:
ah! I see what you mean. Just didn't realize that there was a model `LineItem` particularly for this scenario. I could create a local variable called line items and pass it into the `form` partial and then we could just use `if line_items render..this`, I would just need to add this to the 'admin/orders/edit' view:
```ruby
    %div{"data-hook" => "admin_order_edit_form"}
    # The line_items local var is the addition here
      = render :partial => 'form', :locals => { :order => @order, :line_items => @order.line_items.exists?  }
 ```